### ISSUE TITLE
My 15-minute count: user download page

### ISSUE THREAD

**Issue by DavidRoy** [BiologicalRecordsCentre/ABLE#364]
New download page is required for individual users: https://butterfly-monitoring.net/mydata/samples/downloads

A clone of the scheme co-ordinator download page: https://butterfly-monitoring.net/downloads

Full details to be specified once ElasticSearch extended to include samples lacking occurrences

**Comment by CrisSevilleja** [comment]
yes @DavidRoy this is quite important, at the moment, the users of 15-min counts cannot download their records on the website. I will put priority on this issue

**Comment by DavidRoy** [comment]
repeat of #369
